lib/Runtime/Language/InterpreterLoop.inl

```diff
@@ -166,7 +166,7 @@ Var Js::InterpreterStackFrame::INTERPRETERLOOPNAME()
     }
 
     Assert(this->returnAddress != nullptr);
-    AssertMsg(m_arguments == NULL || Js::ArgumentsObject::Is(m_arguments), "Bad arguments!");
```

Comment:
Looks like with @rajatd's Inlining in jitted loop bodies change, we do have this info on FunctionBody. will update the diff.